targets/liberty-apis/pom.xml

```diff
@@ -19,228 +19,234 @@
       <artifactId>com.ibm.websphere.appserver.api.j2eemanagement</artifactId>
-      <version>1.1.12</version>
+      <version>1.1.13</version>
       <type>jar</type>
     </dependency>
     <dependency>
       <groupId>com.ibm.websphere.appserver.api</groupId>
       <artifactId>com.ibm.websphere.appserver.api.jaxrs</artifactId>
-      <version>1.0.12</version>
+      <version>1.0.13</version>
       <type>jar</type>
     </dependency>
     <dependency>
       <groupId>com.ibm.websphere.appserver.api</groupId>
       <artifactId>com.ibm.websphere.appserver.api.jaxrs20</artifactId>
-      <version>1.0.12</version>
+      <version>1.0.13</version>
       <type>jar</type>
     </dependency>
     <dependency>
       <groupId>com.ibm.websphere.appserver.api</groupId>
       <artifactId>com.ibm.websphere.appserver.api.json</artifactId>
-      <version>1.0.12</version>
+      <version>1.0.13</version>
       <type>jar</type>
     </dependency>
     <dependency>
       <groupId>com.ibm.websphere.appserver.api</groupId>
       <artifactId>com.ibm.websphere.appserver.api.kernel.service</artifactId>
-      <version>1.0.12</version>
+      <version>1.0.13</version>
       <type>jar</type>
     </dependency>
     <dependency>
       <groupId>com.ibm.websphere.appserver.api</groupId>
       <artifactId>com.ibm.websphere.appserver.api.messaging</artifactId>
-      <version>1.0.12</version>
+      <version>1.0.13</version>
       <type>jar</type>
     </dependency>
     <dependency>
       <groupId>com.ibm.websphere.appserver.api</groupId>
       <artifactId>com.ibm.websphere.appserver.api.monitor</artifactId>
-      <version>1.1.12</version>
+      <version>1.1.13</version>
       <type>jar</type>
     </dependency>
     <dependency>
       <groupId>com.ibm.websphere.appserver.api</groupId>
       <artifactId>com.ibm.websphere.appserver.api.oauth</artifactId>
-      <version>1.2.12</version>
+      <version>1.2.13</version>
       <type>jar</type>
     </dependency>
     <dependency>
       <groupId>com.ibm.websphere.appserver.api</groupId>
       <artifactId>com.ibm.websphere.appserver.api.oidc</artifactId>
-      <version>1.0.12</version>
+      <version>1.0.13</version>
       <type>jar</type>
     </dependency>
     <dependency>
       <groupId>com.ibm.websphere.appserver.api</groupId>
       <artifactId>com.ibm.websphere.appserver.api.persistence</artifactId>
-      <version>1.0.12</version>
+      <version>1.0.13</version>
       <type>jar</type>
     </dependency>
     <dependency>
       <groupId>com.ibm.websphere.appserver.api</groupId>
       <artifactId>com.ibm.websphere.appserver.api.restConnector</artifactId>
-      <version>1.1.12</version>
+      <version>1.1.13</version>
       <type>jar</type>
     </dependency>
     <dependency>
       <groupId>com.ibm.websphere.appserver.api</groupId>
       <artifactId>com.ibm.websphere.appserver.api.saml20</artifactId>
-      <version>1.0.12</version>
+      <version>1.1.13</version>

```

Comment:
Yes that's correct:
http://search.maven.org/#search|gav|1|g%3A%22com.ibm.websphere.appserver.api%22%20AND%20a%3A%22com.ibm.websphere.appserver.api.saml20%22